altair/api.py

```diff
@@ -233,6 +293,18 @@ def _shape_changed(self, name, old, new):
             self.shape._infer_type(self.parent.data)
 
 
+class Filter():
```

Comment:
I'm talking from a vega-lite syntax perspective.  

After 0.8 release, the top-level `filter` will become `data.filter`.   
Also, we are using Vega expression instead of the old predicate object.  
Here is the corresponding PR – https://github.com/vega/vega-lite/pull/603.

For other syntax changes in the pending 0.8, please take a look at https://github.com/vega/vega-lite/pull/595.